src/App.js

```diff
@@ -1,18 +1,26 @@
 import './App.css';
```

Comment:
Beginning on line 55 of this file, take a look at what props you can remove from prop drilling. You're setting your state perfectly in context, so there's no need to pass those pieces of state down as props, as well.